### PR TITLE
More workable code

### PR DESCRIPTION
**Pull request by kmcostel**
Create a doc wasn't working in search (home) page, working now.

### CODE REVIEW

File: view.html [hunk 1]
```diff
@@ -273,7 +273,7 @@
 
   <body>
     <h1>MarkLogic <span class="definition" title="Bitemporal: Handles historical data along two different time lines, making it possible to rewind the information “as it actually was” in combination with “as it was recorded” at some point in time">Bitemporal</span> Visualization Demo</h1>
-    <a href="/" id="homeURL">Home</a>
+    <a href="search" id="searchURL">Home</a>
```
Comment (lrhruska) [inline view.html]: should be <a href="/" id="homeURL">Home</a>

File: public/bitemp_graph_helper.js [hunk 2]
```diff
@@ -289,7 +288,7 @@ function saveNewDoc(chart) {
     data: data,
     processData: false,
     success: function(data) {
-      if(!window.location.href.endsWith('/search')) {
+      if(!window.location.href === 'http://localhost:3000/') {
```
Comment (dmcassel) [inline public/bitemp_graph_helper.js]: Don't rely on the server being localhost or the port being 3000 -- what if you're accessing the app running on a different server? what if it's deployed to a different port?

File: search.html [hunk 3]
```diff
@@ -654,7 +654,7 @@
         <textarea rows="10" cols="57" color="blue" id="queryText" readonly></textarea>
       </div>
     </div>
-    <div style="border-left:medium #000080 solid; height:800px; position:absolute; left:600px; top:100px;"/>
+    <div style="border-left:medium #000080 solid; height:800px; position:absolute; left:600px; top:100px;">
```
Comment (dmcassel) [inline search.html]: why drop the closing bracket? Is this div closed farther down?

File: public/bitemp_graph_helper.js (hunk 2 above)
Comment (kmcostel) [inline public/bitemp_graph_helper.js]: Yea that's true, good point

**Pull request merged**